### PR TITLE
github: limit PR search to Homebrew

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-----

Not sure whether this is the way you want to handle this problem but it's really darn irritating so here's a PR that handles it one way.

Fixes/hacks around: https://github.com/Homebrew/brew/pull/3086#issuecomment-324519156

### CODE REVIEW

File: Library/Homebrew/utils/github.rb [hunk 1]
```diff
@@ -245,7 +245,7 @@ def issues_for_formula(name, options = {})
   end
 
   def print_pull_requests_matching(query)
-    open_or_closed_prs = search_issues(query, type: "pr")
+    open_or_closed_prs = search_issues(query, type: "pr", user: "Homebrew")
```
Comment (JCount) [inline Library/Homebrew/utils/github.rb]: An potential alternative would be to use `, org: "Homebrew"` here instead.

**Comment by MikeMcQuaid** [comment]
Nice catch @DomT4, thanks again!

File: Library/Homebrew/utils/github.rb (hunk 1 above)
Comment (DomT4) [reply Library/Homebrew/utils/github.rb]: Let me know if you feel strongly about this & I'll follow-up.

**Comment by DomT4** [comment]
Thanks! It seems to resolve the original problem alright but if anyone reports any weirdness feel free to prod me & I'll look at alternative options 😄.

`brew search goat` now returns a more easily-digested:
```
==> Searching local taps...
==> Searching taps on GitHub...
==> Searching blacklisted, migrated and deleted formulae...
No formula found for "goat".
Closed pull requests:
audit: don't output when searching taps. (https://github.com/Homebrew/brew/pull/3086)
sslscan: add option to statically link OpenSSL (https://github.com/Homebrew/homebrew-core/pull/10556)
qt5: enable all options (https://github.com/Homebrew/homebrew-core/pull/8523)
Update xapian to new version 1.2.18 (https://github.com/Homebrew/legacy-homebrew/pull/32137)
```

**Pull request merged**